include/eve/algo/concepts/nothing.hpp

```diff
@@ -0,0 +1,20 @@
+//==================================================================================================
```

Comment:
I think this can just lives in eve/detail/meta.hpp no ?
Or is it part of the public API ?

Reply:
Well, it could.
Here is my thinking:
it should be public, because the user will see it sometimes.

Like:

`map_range<std::vector<int const>, my_lamda, eve::algo::nothing_t>`

I feel in this case it should not be in detail

Reply:
OK